src/BuildScriptGenerator/Python/PythonBashBuildSnippet.sh.tpl

```diff
@@ -68,6 +86,23 @@ then
 	then
 		exit $pipInstallExitCode
 	fi
+elif [ -e "setup.py" ]
+then
+	echo
+	START_TIME=$SECONDS
+	pip install --upgrade pip
+	ELAPSED_TIME=$(($SECONDS - $START_TIME))
+	echo "Done in $ELAPSED_TIME sec(s)."
+
+	echo "Running python setup.py install..."
+	$python setup.py install --user| ts $TS_FMT
+	cd *.egg-info
```

Comment:
python<version> setup.py install build and installs dependent packages. It installs packages in the userbase (it's needed for build image as in anteras  build env we don't have write permission) and also creates a <projectname>.egg-info folder. Inside that egg-info folder we have the requires.txt. which we do install again using pip (like we do for our normally when we see a requirement.txt in root of the app).